WordPress/src/main/java/org/wordpress/android/ui/prefs/ExperimentalFeaturesActivity.kt

```diff
@@ -132,17 +139,23 @@ fun ExperimentalFeaturesScreen(
     }
 }
 
+@SuppressLint("DiscouragedApi")
```

Comment:
Rather than use a discouraged API here, how about changing the `Feature` class to have the actual label string resource ID?

```
data class Feature(
    val enabled: Boolean = false,
    val key: String,
    @StringRes val label: Int
)
```

Reply:
One benefit of dynamically retrieving the label based on the `key` is that we no longer have to pass the `label` around when mutating the `enabled` state. The discouraged API appears to relate to performance, but I'm not sure it is very relevant for this specific use case. WDYT?

Reply:
We're probably fine to stick with this, I just figured since we previously passed around `label` as a string it wouldn't be a problem as an Int.